src/Jackalope/NodeType/NodeProcessor.php

```diff
@@ -178,13 +185,15 @@ private function processNodeWithType(Node $node, NodeType $nodeTypeDefinition)
                                 $value = $defaultValues;
                             } elseif (isset($defaultValues[0])) {
                                 $value = $defaultValues[0];
-                            } else {
+                            } elseif ($nodeTypeDefinition->getName() !== VersionHandler::MIX_VERSIONABLE) {
```

Comment:
I have stolen this implementation detail from Jackrabbit: http://svn.apache.org/repos/asf/jackrabbit/trunk/jackrabbit-core/src/main/java/org/apache/jackrabbit/core/ItemSaveOperation.java

The versioning information is added at a later point, since consistency checks make it hard (or even impossible?) to add them before or during the check done by the `NodeProcessor`.